src/assets/js/alma-checkout.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Checkout page.
+ *
+ * @package Alma_WooCommerce_Gateway
+ */
+
+(function ($) {
+
+	$( 'body' ).on(
+		'change',
+		'input[name="alma_fee_plan"]',
+		function() {
+			if ( $( this ).prop( 'checked' ) ) {
+				$( this ).closest( 'li.wc_payment_method' ).attr( 'data-already-checked', $( this ).attr( 'id' ) );
+			}
+		}
+	);
+
+	$( 'body' ).on(
+		'payment_method_selected',
+		function(){
+			var payment_method = $( '.woocommerce-checkout input[name="payment_method"]:checked' ).closest( 'li.wc_payment_method' );
+			if (typeof payment_method.attr( 'data-already-checked' ) != 'undefined') {
+				$( '#' + payment_method.attr( 'data-already-checked' ) ).prop( 'checked', true ).trigger( 'change' );
+			} else {
+				let fee_plans        = payment_method.find( 'input[name="alma_fee_plan"]' );
```

Comment:
configure your ide to check javascript with ECMA Script 5.1

`let` is not supported before ECMA Script 6

Reply:
fix this issue please

Reply:
please fix this issue

Reply:
it is now fixed, there is no "let" anymore.